book/src/dev/zebra-dependencies-for-audit.md

```diff
@@ -0,0 +1,150 @@
+# Zebra dependencies
+
+This is a list of Rust code that is in scope and out of scope for Zebra's first audit:
+- **bold**: Crates that we want to be audited fully.
+- <i>italic</i>: Crates we want to be audited with conditions.
+
+## Crates
+
+| Name | Version | Notes
+|------| ------- | -----
+| [**tower-batch**](https://github.com/ZcashFoundation/zebra/tree/main/tower-batch) | [0.2.31](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/tower-batch) |
+| [**tower-fallback**](https://github.com/ZcashFoundation/zebra/tree/main/tower-fallback) | [0.2.31](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/tower-fallback) | 
+| [**zebra-chain**](https://github.com/ZcashFoundation/zebra/tree/main/zebra-chain) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-chain) |
+| [**zebra-consensus**](https://github.com/ZcashFoundation/zebra/tree/main/zebra-consensus) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-consensus) |
+| [**zebra-network**](https://github.com/ZcashFoundation/zebra/tree/main/zebra-network) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-network) |
+| [**zebra-node-services**](https://github.com/ZcashFoundation/zebra/tree/main/zebra-node-services) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-node-services)  |
+| [**zebra-rpc**](https://github.com/ZcashFoundation/zebra/tree/main/zebra-rpc) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-rpc) |
+| [**zebra-script**](https://github.com/ZcashFoundation/zebra/tree/main/zebra-script) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-script) |
+| [**zebra-state**](https://github.com/ZcashFoundation/zebra/tree/main/zebra-state) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-state) |
+| [**zebrad**](https://github.com/ZcashFoundation/zebra/tree/main/zebrad) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebrad) |
+| [<i>zebra-utils</i>](https://github.com/ZcashFoundation/zebra/tree/main/zebra-utils) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-utils) | <i>Only [zebra-checkpoints](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-utils/src/bin/zebra-checkpoints) utility needs to be audited.</i>
```

Comment:
Do we want to use the `Cargo.toml` versions where most of these are `1.0.0-beta.16`?

Reply:
Yes, the crate versions for everything except `zebrad` are incorrect, most Zebra crates are still in beta.

Reply:
Updated in https://github.com/ZcashFoundation/zebra/pull/5471

Reply:
I updated the versions in https://github.com/ZcashFoundation/zebra/pull/5471